ccf_transaction_processor/generate_signing_keys/key_gen.py

```diff
@@ -0,0 +1,46 @@
+# Copyright 2020 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from infra.clients import CCFClient
+import argparse
+
+def run(args):
+    
+    host = args.host
+    port = 6006
+    cert = "./user1_cert.pem"
```

Comment:
here user1 is hardcoded, and there is no mention of tests. Will this require any refactoring later?

Reply:
thank you, I am changing it to user0. No refactoring later. CCF users are named user0, user1 etc. As of now the deployemnt script by default generates 3 CCF users user0/1/2. given that in the documentation, we ask pdo clients to take the role of user0, i am changing these scripts also to use user0 (as noted in an earlier response, we might be able to deploy ccf with exactly one user, but given that in pdo we manage user identities within the TP itself, I am not going to see how to fix this now). Thanks !